package.json

```diff
@@ -10,16 +10,16 @@
     "build:watch": "babel --watch src --out-dir dist",
     "prepare": "not-in-install && npm run build || true",
     "report-coverage": "nyc report --reporter=lcov | codecov",
-    "write-coverage": "nyc report --reporter=lcov",
+    "write-coverage": "nyc report --reporter=lcov --reporter=cobertura --reporter=html",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start": "npm run test:watch",
     "lint": "eslint --ignore-path .gitignore .",
-    "test": "nyc --require @babel/register _mocha -- test/tests/index.js",
+    "test": "nyc --require @babel/register _mocha --reporter=mocha-multi-reporters --reporter-options configFile=./test/mochareporters.json -- test/tests/index.js",
     "test:watch": "nodemon -q --ignore test/.tmp/ --ignore test/artifacts/ --ignore coverage/ --exec \"npm run test\" --"
   },
   "config": {
     "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
+      "path": "./node_modules/cz-conventional-changelog-default-export"
```

Comment:
Hey! This looks pretty good but I was wondering if this line is intentional?

Reply:
@jimthedev Hey. Sorry I missed this conversation. This change wasn't intentional and not related to the working I guess.

Reply:
If you can update it then I can merge.